ehrql/query_language.py

```diff
@@ -70,6 +70,23 @@ def define_population(self, population_condition):
         validate_population_definition(population_condition._qm_node)
         self.variables["population"] = population_condition
 
+    def add_column(self, column_name: str, ehrql_query):
+        """
+        Add a column to the Dataset
```

Comment:
Nitpick!

```suggestion
        Add a column to the dataset.
```

Reply:
I saw that it was capitalised in `define_population` docstring but you're right, more often we use lowercase and I also think it's clearer.